aip/client-libraries/4222.md

```diff
@@ -154,6 +154,31 @@ If none of the routing parameters matched their respective fields, the routing h
 Much like URL parameters, if there is more than one key-value pair to be sent, the `&`
 character is used as the separator.
 
+### `path_template` syntax
+
+As seen in the above examples, the `path_template` can use a variety of symbols
+that are interpreted by code generators during conversion to regular expression.
+The syntax for these symbols is as follows:
+
+- Wildcards:
+  - `*`: Corresponds to 1 or more non-`/` symbols. The regex describing it is
+    `[^/]+`.
+  - `**`: Corresponds to 0 or more symbols or segments (so it can be empty). The
+    regex is `.*`.
+- `{}`: Named segments, the contents of which are used as a value in the
+  key-value pair of the header value. These have one of the following forms:
+  - `{key}`: Where `key` is the name of the key to be used in the key-value pair
+    sent. Defaults to a format `**` for capturing the contents of the value to
+    be paired.
+  - `{key=format}`: Where `format` is the format of the segment(s) (expressed in
+    this `path_template` syntax) to extract as the value paired with `key`.
```

Comment:
Is the format actually recursive?

Reply:
No, I'm trying to convey that a `*` or `**` inside of a `{}` means the same thing as outside of it. I'll add a point about not being able to nest `{}`, but is there another way to word this better?

Reply:
Done. Please resolve if you are satisfied with the new phrasing.

Reply:
LGTM. I don't have permissions to resolve.